README.md

```diff
@@ -205,13 +208,21 @@ Can also be an object formatted like
 
 Where `.myElement` is any valid jQuery selector.
 
+> **required:** `no`
+> **default:** `''`
+
+
 ##### beforeShowPromise
 
 A function that returns a promise. When the promise resolves, the rest of the `show` code for the step will execute. This is a good place to schedule things in the Ember.run loop that you need to ensure happen before show.
 
+> **required:** `no`
```

Comment:
Can we use `required` and `optional` as the descriptors instead? This addon would also be another candidate for ember-cli-addon-docs, where we can make much nicer docs.

Reply:
So instead of "yes" or "no", you're thinking just saying something like:

> **required**

or

> **optional**

Reply:
Yep, exactly or we wouldn't even need to say `optional`, we could just mark `required` ones, but either way.

Reply:
Updated